lua/swenv/api.lua

```diff
@@ -94,6 +94,18 @@ local get_venvs_for = function(base_path, source, opts)
       source = source,
     })
   end
+  -- add base env in conda
```

Comment:
Looks good, only thing I would ask is to move this to a separate function for better separation of concerns. Currently `get_venvs_for` is not specific to any venv type and I think it would be better to add a new line here:
https://github.com/AckslD/swenv.nvim/pull/33/files#diff-ac1a7940439c2242d22b2edf185c48eaf0e2b9ac3f5732a27d702e9dee8429e8L142
with a function (eg `get_conda_base`) that returns a list with the single conda base venv of empty if it doesn't exist. Ie something like
```lua
  vim.list_extend(venvs, get_conda_base())
```

What do you think?

Reply:
Hi Axel, thanks for your review and feedback! I think you're right, separating the code makes a cleaner code structure. I've moved the code to a `get_conda_base_env` function and added a line in `M.get_envs` in the latest commit.

Reply:
Cool thanks!